.github/workflows/release.yml

```
@@ -19,6 +19,9 @@ jobs:
     environment:
       name: npm
       url: https://www.npmjs.com/package/@dfinity/pic
+    permissions:
+      contents: read
```

We need write access to make the action create the GitHub release in the last step

```suggestion
      contents: write
```